components/containers/provider-dashboard/prize-tap/CardTimer.tsx

```diff
@@ -90,7 +90,10 @@ export const ProviderDashboardCardTimer = ({
             <p className="prize-card__timer-item-label text-gray90">s</p>
           </div>
         </div>
-      ) : deadline.getTime() - now.getTime() > 0 ? (
+      ) : (deadline.getTime() - now.getTime() > 0 && Number(days) > 0) ||
+        Number(minutes) > 0 ||
+        Number(hours) > 0 ||
+        Number(seconds) > 0 ? (
```

Comment:
provide more description of the evalutation by declaring this expression into a variable